test/localIntegrationRunner.html

```diff
@@ -17,12 +17,7 @@
       mocha.ui('bdd')
     </script>
 
-    <script src="../testCompiledForWeb/offerCancel.js"></script>
-    <script src="../testCompiledForWeb/offerCreate.js"></script>
-    <script src="../testCompiledForWeb/payment.js"></script>
-    <script src="../testCompiledForWeb/signerListSet.js"></script>
-    <script src="../testCompiledForWeb/utility.js"></script>
-    <script src="../testCompiledForWeb/integration.js"></script>
+    <script src="../testCompiledForWeb/index.js"></script>
 
```

Comment:
Did you mean to delete these integration tests? (I think the expected output for this file should be to have all the integration tests in here as <script> ... </script> files)